### DISCUSSION TITLE
Upgrading the Scribus Appimage creation to use Qt6

### DISCUSSION THREAD

**Discussion by aoloe** [AppImage/AppImageKit#1329] [Q&A]
I've now sorted out most issues with compiling Scribus 1.7svn (with Qt6) and creating the Appimage.

The last remaining bits are:

```
ERROR: findDependencyInfo: "\tnot a dynamic executable\n"
ERROR: getBinaryRPaths: "objdump: /builds/scribus/scribus/appdir/usr/share/scribus/translations/scribus.so.qm: file format not recognized\n"
ERROR: Error stripping "libz.so" : ""
ERROR: Error stripping "libz.so" : ""
```

This happens when running:

```
ARCH=x86_64 VERSION=$SCRIBUS_VERSION ./linuxdeployqt-continuous-x86_64.AppImage --appimage-extract-and-run appdir/usr/share/applications/scribus.desktop \
                                               -qmake=`which qmake6` \
                                               -appimage -unsupported-bundle-everything \
                                               -executable=appdir/usr/bin/python3.11 \
                                               -executable=appdir/usr/lib/python3.11/lib-dynload/_tkinter.cpython-311-x86_64-linux-gnu.so \
                                               -extra-plugins=platformthemes/libqgtk2.so,styles/libqgtk2style.so
```

Here are the relevant files:

- Here is the <https://gitlab.com/scribus/scribus/-/blob/CI-17/AppImage-package/bundle.sh?ref_type=heads>
- <https://gitlab.com/scribus/scribus-ci-docker/-/blob/main/Dockerfile?ref_type=heads>

Does anybody have an idea, how i can get rid of this remaining errors?

**Comment by probonopd** [comment]
Hello @aoloe. Well, to begin with, never use `-unsupported-...` flags. They are broken, and they likely result in broken AppImages. If you are looking for this kind of functionality, then maybe go-appimage's appimagetool `-s deploy` feature is suitable?
